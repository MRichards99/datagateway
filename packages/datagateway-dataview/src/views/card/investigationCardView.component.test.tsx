import { createShallow } from '@material-ui/core/test-utils';

describe('Investigation - Card View', () => {
  // remove following line once tests are created
  //eslint-disable-next-line @typescript-eslint/no-unused-vars
  let shallow;

  beforeEach(() => {
    shallow = createShallow();
  });

  it.todo('renders correctly');
});
