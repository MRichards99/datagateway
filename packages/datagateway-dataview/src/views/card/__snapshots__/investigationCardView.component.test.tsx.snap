// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Investigation - Card View renders correctly 1`] = `
<ContextProvider
  value={null}
>
  <InvestigationCardView
    addToCart={[Function]}
    cartItems={Array []}
    data={
      Array [
        Object {
          "FACILITY_ID": "2",
          "ID": 1,
          "NAME": "Test 1",
          "TITLE": "Test 1",
          "TYPE_ID": "2",
          "VISIT_ID": "1",
        },
      ]
    }
    fetchCount={[Function]}
    fetchData={[Function]}
    fetchFacilityFilter={[Function]}
    fetchTypeFilter={[Function]}
    filterData={Object {}}
    loadedCount={true}
    loadedData={true}
    pushFilters={[Function]}
    pushPage={[Function]}
    pushQuery={[Function]}
    query={
      Object {
        "filters": Object {},
        "page": null,
        "results": null,
        "search": null,
        "sort": Object {},
        "view": null,
      }
    }
    removeFromCart={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    totalDataCount={1}
  />
</ContextProvider>
`;
